### ISSUE TITLE
friction: PPO-Mamba dry-run still segfaults in optional readiness

### ISSUE THREAD

**Issue by ll7** [ll7/robot_sf_ll7#5724]
## Observed failure

The final local `BASE_REF=origin/main scripts/dev/pr_ready_check.sh` optional lane also aborts on the PPO-Mamba dry-run test with a native signal:

```text
tests/training/test_train_ppo_mamba_config_issue_4014.py::test_ppo_mamba_dry_run_emits_parameter_summary_placeholder
CompletedProcess.returncode == -11
```

The focused test reproduces the `-11` serially in the current isolated gate environment, so this is not only an xdist scheduling artifact. The prior local native-crash tracker [#5633](https://github.com/ll7/robot_sf_ll7/issues/5633) was closed after an earlier readiness-path repair; this is new evidence that the optional PPO-Mamba smoke remains unsafe on the current dependency/runtime combination.

## Scope

- Identify the native extension/runtime combination causing the PPO-Mamba dry-run signal 11.
- Make the supported local readiness path fail cleanly or use a safe compatibility guard without silently skipping the test.
- Preserve the diagnostic claim boundary for issue #4014.

## Acceptance

- The focused PPO-Mamba dry-run completes successfully or emits an actionable fail-closed availability result rather than signal 11.
- The optional readiness lane records the outcome without a native worker crash.

## Evidence

- Current main: `17b88892311f4f6737d6b08380c52825367249d4`
- Reproduction: `uv run pytest tests/training/test_train_ppo_mamba_config_issue_4014.py::test_ppo_mamba_dry_run_emits_parameter_summary_placeholder -q`
- Failure is outside PR #5714's changed files and is linked from its gate findings.

**Comment by ll7** [comment]
Merged #5739 (squash 695b576c787acb41ddd7834eb4aba4649df82f3e): the conflict guard now covers TensorFlow, TensorBoard, and Keras imports, with 11 seed-guard tests and 3 PPO-Mamba dry-run tests passing; no #5724 acceptance criteria remain.
